src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemoveView.tsx

```diff
@@ -23,9 +23,52 @@ export function AutomaticRemoveView(props: Props) {
 
   const dataBrokerCount = CONST_ONEREP_DATA_BROKER_COUNT;
 
-  const { monthlySubscriptionUrl, yearlySubscriptionUrl, ...fixViewProps } =
-    props;
+  let { monthlySubscriptionUrl, yearlySubscriptionUrl } = props;
+  const { ...fixViewProps } = props;
```

Comment:
I think the destructuring was there to remove the `monthlySubscriptionUrl` and `yearlySubscriptionUrl` props from `fixViewProps`. The destructuring is now a bit idiosyncratic, because `viewFixProps` is essentially just a clone of `props` - why not just pass that then?

As possible refactoring could be to instead just create new local variables (e.g. `normalisedMonthlySubscriptionUrl`) and modify _those_ rather than changing the function's input parameters.